packages/nextjs/components/SwitchTheme.tsx

```diff
@@ -29,7 +29,7 @@ export const SwitchTheme = ({ className }: { className?: string }) => {
       <input
         id="theme-toggle"
         type="checkbox"
-        className="toggle toggle-primary bg-primary hover:bg-primary border-primary"
+        className="toggle bg-base-content focus:bg-base-content border-base-content"
```

Comment:
new:
![image](https://github.com/user-attachments/assets/fdb809df-70aa-4588-844f-cb1a40deede8)
old:
![image](https://github.com/user-attachments/assets/5c8fb59f-e83b-4363-a124-e218e1d5affc)

Maybe the switch button looks a little bit too pronounced with these changes?

Reply:
Yes I agree, but It will be difficult for some extensions themes to make all the colors look good in both light and dark theme. But if to make the color `base-content` we are sure that colors will be distinguishable. I can add for example 70-90 opacity, but not sure we need it here

Reply:
> Maybe the switch button looks a little bit too pronounced with these changes?

I agree, I think maybe we should revert this changes? 

Its fine if colors don't go nicely in extensions but we make sure by default base se-2 looks nice. 

I just tried previous changes to SRE base template and it looks fine : 
![Screenshot 2024-11-21 at 4 26 37 PM](https://github.com/user-attachments/assets/64916522-43bd-4ddb-a495-e04928be2c59)
![Screenshot 2024-11-21 at 4 26 54 PM](https://github.com/user-attachments/assets/6fb1b210-216b-4061-900c-84e5214a801a)

Lol light mode doesn't look that great but I think for extensions its fine also non-deterministic what people gonna put in their tailwind.config.js template, its better we optimize for base SE-2?

Reply:
Reverted [2113370](https://github.com/scaffold-eth/scaffold-eth-2/pull/991/commits/21133704253ddec64e966c6f534bfe64bc96fb92), but I leave some thougths regarding it

> Its fine if colors don't go nicely in extensions but we make sure by default base se-2 looks nice.

> Lol light mode doesn't look that great but I think for extensions its fine also non-deterministic what people gonna put in their tailwind.config.js template, its better we optimize for base SE-2

The problem is that sre will be group of the most used (or one of the most used) extensions and sre users are important for us. Also, I think most of the users use default light theme. I think we need to make it as good as we can. 

For me, the changed version looks a bit worse than current, but still ok. Well, if not count that I broke hover part a bit :). Since toggle color in that version is the same as color of text, it will work by default with most of the extensions

---

As a note, we can still change a bit sre default colors so maybe we will make it better without changing toggle inside se-2

Reply:
> As a note, we can still change a bit sre default colors so maybe we will make it better without changing toggle inside se-2

Ohh yes, I was thinking that people could always tweak their tailwind theme since its non-deterministic which we can't handle. 

> The problem is that sre will be group of the most used (or one of the most used) extensions. Also, I think most of the users use default light theme.
> For me, the changed version looks a bit worse than current, but still ok. Well, if not count that I broke hover part a bit :). Since toggle color in that version is the same as color of text, it will work fine by default with most of the extensions

Yup, I agree, maybe let's merge this if we don't find a way to tweak the challenge color and make it look better we can come back to this discussion on SE-2 and tweak SE-2 color palette a bit so that it suits all extensions as well 🙌 , Thanks Rinat!!